### PR TITLE
add contact form.

### PR DESCRIPTION
**Pull request by brahimdidi**
## In this Project I will create a contact form.

Requirements:
Add a contact form to your portfolio and the portfolio of your coding partner.
Make sure that you guide your users' experience in a way that helps them fill in the form without frustration.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -175,16 +175,16 @@ <h5 class="h5">CONTACT US</h5>
...
                 <label for="name"></label>
-                <input type="text" name="name" id="name" class="formname" placeholder="Text input" maxlength="20" required>
+                <input type="text" name="name" id="name" class="formname" placeholder="text" maxlength="30" required>
                 <label for="email"></label>
-                <input type="email" name="email" id="email" class="formemail" placeholder="brahimdidi@gmail.com" maxlength="30" required>
+                <input type="email" name="email" id="email" class="formemail" placeholder="text" maxlength="30" required>
             
-            <textarea name="message" class="formmessage" id="message"  maxlength="400" placeholder="Enter text here" required></textarea>
-            <button class="seeproject2" type="submit">See Project</button>
+            <textarea name="message" class="formmessage" id="message"  maxlength="500" placeholder="text" required></textarea>
```
Comment (AshakaE) [inline index.html:185]: - [ ] Please consider using an appropriate placeholder for the input fields and textarea, I suggest `name` , `name@email.com`, and `enter your message` respectively 🙂

File: index.html [hunk 2]
```diff
@@ -175,16 +175,16 @@ <h5 class="h5">CONTACT US</h5>
...
             </div>
+            <div class="btn2div"><button class="seeproject2" type="submit">See Project</button></div>
```
Comment (AshakaE) [inline index.html:187]: - [ ] Please consider using an appropriate name for the button either `submit` or `send`

**Review by AshakaE — CHANGES_REQUESTED** [review]
### Hi @brahimdidi  🙋‍♀️, 

Good job so far! 👌
There are some issues that you still need to work on to go to the next project but you are almost there!

#  Required Changes ♻️ 🚧
_Check the comments under the review._

###  Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (brahimdidi) [reply index.html:185]: it is required to be text

File: index.html (hunk 2 above)
Comment (brahimdidi) [reply index.html:187]: It is see project in the figma They said we should stick to the design

**Review by Murodjon000 — APPROVED** [review]
**Hi Didi  👋,** 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

- Please capitalize your `submit` button before merging. It is to do so that reason I will approve. UI is perfect😊😊

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**